src/Util/Lifecycle/ActivateDeactivate.php

```diff
@@ -37,13 +37,12 @@ public function activate(Context $context): void
     {
         $this->paymentMethodStateService->setAllPaymentMethodsState(true, $context);
         $this->posStateService->addPosSalesChannelType($context);
+        $this->posStateService->setPosSalesChannelState(true, $context);
```

Comment:
As we cannot differentiate between deactivations on purpose and ones done by us, just leave them deactivated